.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## What's new
+
+fixes #
+
+[Describe your changes]
+
+## Self-checks
+
+### Frontend
+
+- [ ] I'm familiar with and follow this [ Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
+- [ ] I added tests for new components
+- [ ] I tried testing edge cases
+
+### Backend connection
+
+- [ ] I'm familiar with and follow this [ Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
+- [ ] I separated the connection logic in a specific file
```

Comment:
Can you explain what does this mean?

Reply:
Yeah, maybe a better name is required for this. 

The main idea of this is taking the logic of the  `negotiation-status-manager` as a example

Reply:
> I separated the connection logic (publisher/subscriber managers) in a specific file

I feel this may be too confusing for new contributors to understand, do you think this is important enough to be on the checklist?

Reply:
Yeah, you're right, we can remove it. Thanks for the feedback.

Reply:
I will look like this then

## What's new

fixes #

[Describe your changes]

## Self-checks

- [ ] I'm familiar with and follow this [ Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

## Discussion

[Questions for reviewers]